orion/sites/admin/src/announce/announce.js

```diff
@@ -3,11 +3,66 @@ require('./announce.styl');
 import React from 'react';
 import ReactDOM from 'react-dom';
 
+
 export class AnnouncePage extends React.Component {
 	render() {
+		var numAnnouncements = 5; 	{/* this needs to be a variable later */}
```

Comment:
ah i see. So inside the `render()` function, comments are still the normal javascript way with `//` or `/* */` You don't need the {}.